src/mono/mono/mini/mini-ppc.c

```diff
@@ -1179,23 +1181,38 @@ get_call_info (MonoMethodSignature *sig)
 				} else
 #endif
 				{
-					align_size += (sizeof (target_mgreg_t) - 1);
-					align_size &= ~(sizeof (target_mgreg_t) - 1);
-					nregs = (align_size + sizeof (target_mgreg_t) -1 ) / sizeof (target_mgreg_t);
-					n_in_regs = MIN (rest, nregs);
-					if (n_in_regs < 0)
-						n_in_regs = 0;
+					if (is_all_floats && mbr_cnt > 0)
+					{
+						rest = PPC_LAST_ARG_REG - gr + 1;
```

Comment:
mono C files use the mono coding style, not the dotnet one.

Reply:
done